fix build with apr 1.3.2 and later
http://www.mail-archive.com/dev@apr.apache.org/msg20588.html

poeml, Tue Aug 19 13:10:20 CEST 2008

diff -uNr mod_python-3.3.1.orig/src/connobject.c mod_python-3.3.1/src/connobject.c
--- mod_python-3.3.1.orig/src/connobject.c	2006-12-03 05:36:37.000000000 +0100
+++ mod_python-3.3.1/src/connobject.c	2008-08-19 13:09:26.831976000 +0200
@@ -139,7 +139,7 @@
     bytes_read = 0;
 
     while ((bytes_read < len || len == 0) &&
-           !(b == APR_BRIGADE_SENTINEL(b) ||
+           !(b == APR_BRIGADE_SENTINEL(bb) ||
              APR_BUCKET_IS_EOS(b) || APR_BUCKET_IS_FLUSH(b))) {
 
         const char *data;
